css-four-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  background-color: rgb(247 248 250);
+  margin-left: -40px;
+  margin-right: -40px;
+  padding-left: 100px;
+  padding-right: 100px;
```

Comment:
`row` should be `display: flex` and nothing else. Again, you need to review `css-layout-classes`.